M/msquic/bundled/patches/msquic.patch

```diff
@@ -0,0 +1,70 @@
+diff --git before/libxml2-2.9.10/include/libxml/xmlversion.h after/libxml2-2.9.10/include/libxml/xmlversion.h
```

Comment:
This patch doesn't look relevant

Reply:
thanks again for the comments and suggestions

we are taking the long route with the dependencies...
If you want all dependencies to be added as jll packages, it will have to be done sequentially, since they depend on each other
I have submitted a pr for urcu (userspace-rcu)
after it is accepted (hopefully soon), I can submit a pr for lttng-ust
and finally after that, a pr for lttg-tools

then, I hope we could address the build problems of msquic

btw, since the wizard accepts external source libraries, I thought that was permissible. If only jll packages are accepted, then perhaps the wizard should be updated accordingly